test/topology/util.py

```diff
@@ -218,49 +218,60 @@ async def wait_until_last_generation_is_in_use(cql: Session):
     else:
         logger.info(f"The last generation is already in use.")
 
-async def check_system_topology_and_cdc_generations_v3_consistency(manager: ManagerClient, hosts: list[Host], cqls: Optional[list[Session]] = None):
+async def check_system_topology_and_cdc_generations_v3_consistency(manager: ManagerClient, live_hosts: list[Host], cqls: Optional[list[Session]] = None, ignored_hosts: List[Host] = []):
```

Comment:
nit: mixing `list` and `List`, should stay consistent and use `list` (the more modern option)

Reply:
is `ignored_hosts` the correct name now? You're not really ignoring them -- you actually verify that they are in "left" state

(and the commit title/message might require updating too)

Reply:
> is `ignored_hosts` the correct name now? You're not really ignoring them -- you actually verify that they are in "left" state

Not exactly. I'm not verifying that every node from `ignored_hosts` is in left state, I'm only verifying that if a node is in a left state then it will be in `ignored_hosts`. It might happen for example that the orphaning process doesn't put a node into the `system.topology` table with a "left" state quickly enough.

Reply:
Right. make sense

Reply:
Updated the commit message to explain the changes that were made to the existing test functions.